staff/acct/sorry/sorry

```diff
@@ -16,6 +17,9 @@ fi
 
 if [ -z "$1" ] || [ -z "$2" ]; then
     echo "Usage: $0 [user to be sorried] [sorry reason file]"
+    echo
+    echo "Sorry Reasons:"
+    find "$REASONPATH" -maxdepth 1 -mindepth 1 -type f -printf "   %f\n"
```

Comment:
I'd also add e.g. "you can also use a custom sorry reason by passing in your own file" or something like that